Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -31,8 +31,9 @@ PRODUCT_BUNDLE_IDENTIFIER = com.apple.$(PRODUCT_NAME:rfc1034identifier);
 INFOPLIST_FILE = Info.plist;
 
 DEFINES_MODULE = YES;
-CLANG_MODULES_ENABLE_VERIFIER_TOOL = $(CLANG_MODULES_ENABLE_VERIFIER_TOOL_$(USE_INTERNAL_SDK));
-CLANG_MODULES_ENABLE_VERIFIER_TOOL_YES = $(WK_NOT_$(WK_XCODE_VERSION_BEFORE_14_3_$(XCODE_VERSION_MAJOR)));
+ENABLE_MODULE_VERIFIER = $(USE_INTERNAL_SDK);
```

Comment:
We no longer need to check for Xcode 14.3?

Reply:
Yeah, these build settings were renamed in Xcode 14.3, so by switching to the new names we are matching existing behavior and keeping the verifier enabled for >= 14.3 only.

Reply:
May be worth adding a comment, as people could be surprised to not see it run despite `ENABLE_MODULE_VERIFIER = YES`.